src/coll/options.rs

```diff
@@ -577,6 +577,11 @@ pub struct DistinctOptions {
 #[derive(Debug, Default, TypedBuilder, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct FindOptions {
+    /// Enables writing to temporary files. When set to true, the find operation can write data to
```

Comment:
I think we should clarify that this allows the _server_ to write to temporary files. I could imagine a user reading this as the driver buffering results on disk. We should also mention that this option is only supported in server versions 4.4+.